snap/snapcraft.yaml

```diff
@@ -0,0 +1,55 @@
+name: gateway
+version: git
+summary: Things Gateway.
+description: Web of Things gateway.
+
+confinement: strict
+grade: stable
+base: core18
+
+apps:
+  gateway:
+    command: env TMPDIR=$XDG_RUNTIME_DIR desktop-launch $SNAP/gateway
+    plugs:
+      - desktop
+      - desktop-legacy
+      - home
+      - x11
+      - browser-support
+      - network
+      - gsettings
+      - pulseaudio
+      - opengl
+
+parts:
+  gateway:
+    plugin: nodejs
+    node-engine: "12.4.0"
```

Comment:
None of our add-ons are currently built for Node 12. Our images right now use Node 8, so it'd be great if we could stick with that for the time being. That can be changed when #2113 is resolved.

Reply:
I was not sure what of node version was being used so thank you for me know I will that fix :+1: